import-export-cli/integration/testdata/TestArtifactDirectory/HiddenFiles/.hidden

```diff
@@ -0,0 +1 @@
+Hidden file
```

Comment:
Instead of storing a hidden file in codebase, can't we create a file using go code as mentioned in [1] by renaming it to something like .hidden, so it will be a hidden file?

[1] https://gosamples.dev/create-file/#:~:text=To%20create%20a%20new%20empty,removed%20without%20deleting%20the%20file.

Reply:
Fixed via [f38280d](https://github.com/wso2/product-apim-tooling/pull/955/commits/f38280d371fd9ce883e2c19f84c1ed8cfe9f1492)

Reply:
@BLasan can't we now remove this hidden file?